src/main/java/io/lettuce/core/CompositeRedisCommandListener.java

```diff
@@ -0,0 +1,37 @@
+package io.lettuce.core;
+
+import io.lettuce.core.models.events.CommandFailedEvent;
+import io.lettuce.core.models.events.CommandStartedEvent;
+import io.lettuce.core.models.events.CommandSucceededEvent;
+
+import java.util.List;
+
+/**
+ * @author sokomishalov
+ */
+public class CompositeRedisCommandListener implements RedisCommandListener {
```

Comment:
Let's rename this class to `CommandListenerMulticaster`

Reply:
done